.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -39,16 +42,9 @@ jobs:
         run: tar -xvf ttm_${{ matrix.arch }}.tar
       - uses: ./.github/actions/install-python-deps
       - name: Run ttnn sweeps
-        timeout-minutes: 30
+        timeout-minutes: 720
```

Comment:
720?

Reply:
12 hours, we will need more than this probably too. There is a lot of tests, at this point just the matmuls and a couple eltwise ops take 4.5 hours to run.

Reply:
That's tough - didn't know it was that much

Please put `timeout-minutes` at the job level, rather than the step level. If you put it at the step level as you do here, then it will be ignored. Any number you put for `steps.<step>.timeout-minutes:` that's higher than 360 will be ignored.

Reply:
Moved to job level.